protocol/x/clob/e2e/withdrawal_gating_test.go

```diff
@@ -350,3 +351,170 @@ func TestWithdrawalGating_BlocksThenUnblocks(t *testing.T) {
 		})
 	}
 }
+
+func TestWithdrawalGating_ChainOutage_BlocksThenUnblocks(t *testing.T) {
+	tests := map[string]struct {
+		// State.
+		subaccount satypes.Subaccount
+
+		// Parameters.
+		secondsBetweenBlocks uint32
+
+		// Configuration.
+		isWithdrawal bool
+
+		// Expectations.
+		expectedWithdrawalsGated bool
+		expectedErr              string
+	}{
+		`Over 5 minutes passes between blocks and withdrawals are gated after the chain restarts`: {
+			subaccount: constants.Dave_Num1_10_000USD,
+
+			secondsBetweenBlocks: 301,
+
+			isWithdrawal: true,
+
+			expectedWithdrawalsGated: true,
+			expectedErr:              "WithdrawalsAndTransfersBlocked: failed to apply subaccount updates",
+		},
+		`Over 5 minutes passes between blocks and transfers are gated after the chain restarts`: {
+			subaccount: constants.Dave_Num1_10_000USD,
+
+			secondsBetweenBlocks: 400,
```

Comment:
planning to also add some quick tests that transfers / withdrawals aren't blocked when less than this timeframe passes.